src/commands/synthetics/__tests__/fixtures.ts

```diff
@@ -275,3 +291,121 @@ export const getSyntheticsProxy = () => {
 
   return {calls, close, config, server}
 }
+
+export interface RenderResultsTestCase {
+  description: string
+  expected: {
+    exitCode: 0 | 1
+    summary: Summary
+  }
+  failOnCriticalErrors: boolean
+  failOnTimeout: boolean
+  fixtures: {
+    results: Record<string, PollResult[]>
+    tests: Test[]
+    triggers: Trigger
+  }
+  summary: Summary
+}
+
+interface RenderResultsTestFixtureConfigs {
+  configOverride: ConfigOverride
+  executionRule?: ExecutionRule
+  publicId: string
+  resultError?: string
+  resultIsUnhealthy?: boolean
+  resultPassed: boolean
+}
+
+interface RenderResultsTestFixtures {
+  polledResultsByPublicId: Record<string, PollResult[]>
+  test: Test
+  triggerResult: TriggerResponse
+}
+
+export class RenderResultsHelper {
```

Comment:
nit: Why use a class here, only `createFixtures()` seems to be called?  We can just use a global functions and a variable for `resultIdCounter` if there is no object state

Reply:
Initially that's what I did, but I thought it would be nicer to have some kind of "namespace" for this particular type of fixtures (hence the class as the namespace).